app/controllers/books_controller.rb

```diff
@@ -52,7 +52,7 @@ def destroy
     @book.destroy
 
     respond_to do |format|
-      format.html { redirect_to books_url, notice: 'Book was successfully destroyed.' }
+      format.html { redirect_to books_url(locale: params[:locale]), notice: t('flash.books.delete') }
```

Comment:
```suggestion
      format.html { redirect_to books_url, notice: t('flash.books.delete') }
```

「毎回忘れずにlocaleをURLに含める」という実装にすると、必ずいつか忘れますし、コードも無駄にボリュームが増えます。
こういうケースはcookieなどにlocaleを保存することが多いですが、課題の対象外なので対応しなくても大丈夫です。
とりあえず、URLにlocaleを含めている箇所は全部なくしていってください〜。